src/main/java/edu/pdx/cs/sftp/User.java

```diff
@@ -4,45 +4,51 @@
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 
-/** User class for SFTP project. Used to gather information about a connection */
+/** Gather information about a connection initiated by the user with the User class */
 class User {
 
-  /** Field for password string */
-  String password;
-  /** Field for username string */
+  /** Store the username in this variable */
   String username;
-  /** Field for host name string */
+  /** Store the password in this variable */
+  String password;
+  /** Store the hostname in this variable */
   String hostname;
-  /** Field for an IO scanner used for input */
+  /** Store IO scanner in this variable */
   private Scanner scanner = new Scanner(System.in);
 
-  /** Default constructor initializes all fields to null */
+  /** Initialize all fields to null with the default constructor */
   User() {
+    username = null;
     password = null;
     hostname = null;
-    username = null;
   }
-
-  User(String pw, String hn, String un) {
-    password = pw;
-    hostname = hn;
-    username = un;
+  /**
+   * Set the User fields to the current input with this constructor
+   *
+   * @param username sets username to current username
+   * @param password sets password to current password
+   * @param hostname sets hostname to current hostname
+   */
+  User(String username, String password, String hostname) {
+    this.username = username;
+    this.password = password;
+    this.hostname = hostname;
   }
 
   /**
-   * Constructor sets scanner for testing purposes
+   * Set the scanner up for testing with this constructor
    *
-   * @param scannerArg String with args for testing
+   * @param scannerArg String with arguments used for testing
    */
   User(String scannerArg) {
+    username = null;
     password = null;
     hostname = null;
-    username = null;
     scanner = new Scanner(scannerArg);
   }
 
   /**
-   * Prompts the user for a valid password
+   * Prompt the user for a valid password
    *
    * @return The password input
```

Comment:
```suggestion
   * @return the password input
```
[nit] this rogue capital letter is triggering my OCD, ha.